WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewModel.kt

```diff
@@ -25,6 +25,8 @@ class StatsViewModel
     @Named(UI_SCOPE) private val uiScope: CoroutineScope,
     @Named(DEFAULT_SCOPE) private val defaultScope: CoroutineScope
 ) : ViewModel() {
+    private lateinit var site: SiteModel
+
     private val _listState = MutableLiveData<StatsListState>()
     val listState: LiveData<StatsListState> = _listState
```

Comment:
I'd have this as a `Boolean` of `isRefreshing`. I think the parent model doesn't care about the children' list state. It only handles the pull to refresh and triggers refresh on all the children.